CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
 # vscode-powershell Release History
 
+## v2.0.0-preview.1
+### Wednesday, January 23, 2019
+
+#### Preview builds of the PowerShell extension are now available in VSCode
+
+We are excited to announce the PowerShell Preview extension in the VSCode marketplace!
+The PowerShell Preview extension allows users on Windows PowerShell 5.1 and PowerShell Core 6 to get and test the latest updates
+to the PowerShell extension and comes with some exciting features.
+
+The PowerShell Preview extension is a substitute for the PowerShell extension so
+both the PowerShell extension and the PowerShell Preview extension should not be enabled at the same time.
+
+By having a preview channel, which only supports Windows Powershell 5.1 and PowerShell Core 6,
```

Comment:
I don't think this needs the `which only supports...` clause. Taking it out makes the statement much more cohesive I think.

Reply:
```suggestion
By having a preview channel,
```

Reply:
fixed